### PR TITLE
[SWE-1199] [BRANDBOOK] GreetingScreen #wip

### PR DESCRIPTION
**Pull request by AnastasiaShapowalova**
Screenshot 2022-07-19 at 21 36 31

### CODE REVIEW

**Comment by AnastasiaShapowalova** [comment]
1 Merged with development, resolved conflicts
2 Fix Header component
3 Add footer to screen
4 Add rectangle

**Comment by AnastasiaShapowalova** [comment]
1 Reduced Img component
2 Add vars to Header component
3 Add params

**Review by olenashevchuk — CHANGES_REQUESTED** [review]

File: src/App.js [hunk 1]
```diff
@@ -1,5 +1,15 @@
+import Greeting from 'assets/hello.png'
```
Comment (olenashevchuk) [inline src/App.js:1]: not a suitable name for image

File: src/components/layouts/GreetingScreen/GreetingScreen.styled.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  line-height: var(--brandbook-module-greeting-screen-title-line-height);
+`
+
+const TextStyled = styled(Text)`
+  font-size: var(--brandbook-module-greeting-screen-text-font-size);
+  line-height: var(--brandbook-module-greeting-screen-text-line-height);
+  margin-left: var(--brandbook-module-greeting-screen-text-margin-left);
+`
+
+const RectangleStyled = styled(Text)`
```
Comment (GingerBoy14) [inline src/components/layouts/GreetingScreen/GreetingScreen.styled.js:36]: what with naming, what mean rectangle = styled(**Text**)...

**Review by GingerBoy14 — CHANGES_REQUESTED** [review]

**Comment by AnastasiaShapowalova** [comment]
1 Delete image component and make styled for image in GreetingScreen
2 Add alt to image
3 Fix ReactangleStyled
4 Fix image naming

File: src/components/layouts/GreetingScreen/GreetingScreen.js [hunk 3]
```diff
@@ -24,11 +24,14 @@ const GreetingScreen = (props) => {
...
-    `${title.slice(0, -1)}<span class='text-accent-color'>${
-      title.slice(-1)[0]
-    }</span>`
+  const titleComputed = useMemo(
+    () =>
+      title &&
+      `${title.slice(0, -1)}<span class='text-accent-color'>${
+        title.slice(-1)[0]
+      }</span>`,
+    []
```
[the remark this answers is not in the document]
Comment (AnastasiaShapowalova) [reply src/components/layouts/GreetingScreen/GreetingScreen.js:33]: not one warnings in console

**Review by olenashevchuk — APPROVED** [review]

**Pull request merged**